target/linux/airoha/an7581/config-6.6

```diff
@@ -232,7 +230,6 @@ CONFIG_FUNCTION_ALIGNMENT_4B=y
 CONFIG_FWNODE_MDIO=y
 CONFIG_FW_CACHE=y
 # CONFIG_FW_LOADER_USER_HELPER is not set
-CONFIG_GCC10_NO_ARRAY_BOUNDS=y
```

Comment:
Please remove `GCC11_NO_ARRAY_BOUNDS` too from all files.
`GCC11_NO_ARRAY_BOUNDS` was renamed to `CONFIG_GCC10_NO_ARRAY_BOUNDS` in https://github.com/gregkh/linux/commit/a5b60c8b9a9e602c83f9e7a714c690cb9acdba61

Reply:
Maybe add this as a separate commit

Reply:
Done.